client/src/components/Workbench/WelcomeMessages.js

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
```

Comment:
FYI: This PR also refactors welcome messages: 
https://github.com/TACC/Frontera-Portal/pull/236